docs/output.md

```diff
@@ -171,4 +171,14 @@ osv-scanner --format json -L path/to/lockfile > /path/to/file.json
 }
 ```
 
-</details>
\ No newline at end of file
+</details>
+
```

Comment:
Please remove changes in this file in favor of #375

Reply:
Undid this change